jobserver/templates/base.html

```diff
@@ -50,6 +50,23 @@
       </div>
     </nav>
 
+    <div class="d-flex justify-content-around align-items-center border-bottom mb-2">
+      <small>
+        Runner Last Seen: {{ site_stats.last_seen }}
+
+        {% if site_stats.show_warning %}
+        <span class="text-warning">⚠️</span>
```

Comment:
Can we add a "tool tip" here saying something like "job runner not seen for at least 5 mins" or similar?